scout/adapter/mongo/case.py

```diff
@@ -1028,12 +1033,15 @@ def update_case(self, case_obj, keep_date=False):
         for key in [
```

Comment:
Could this one be collected from the constants instead? So we avoid hardcoding every new key